docs/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -10,154 +12,217 @@
+    - [ ] Publish the release to [the NPM package](https://www.npmjs.com/package/go-ipfs?activeTab=versions) by running https://github.com/ipfs/npm-go-ipfs/actions/workflows/main.yml (it happens automatically but it is safe to speed up the process and kick of a run manually)
+  - [ ] Cut the release on [GitHub](https://github.com/ipfs/kubo/releases) ([instructions](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release), [example](https://github.com/ipfs/kubo/releases/tag/v0.16.0))
+    - Use `vX.Y.Z` as the tag.
+    - Link to the release issue in the description.
+    - Copy the relevant [changelog](https://github.com/ipfs/kubo/blob/release/docs/changelogs/) into the release description.
+      - Keep the release notes as trim as possible (e.g. remove top headers where possible, [example](https://github.com/ipfs/kubo/releases/tag/v0.15.0))
+  - [ ] Synchronize release artifacts by running [sync-release-assets](https://github.com/ipfs/kubo/actions/workflows/sync-release-assets.yml) workflow.
+  - [ ] Announce the release
+    - [ ] Add a link to the release to this release issue as a comment.
+    - [ ] Create a new post on [IPFS Discourse](https://discuss.ipfs.tech). ([example](https://discuss.ipfs.tech/t/kubo-v0-16-0-release-is-out/15286))
+      - Use `Kubo vX.Y.Z Release is out!` as the title.
+      - Use `kubo` and `go-ipfs` as topics.
+      - Repeat the title as a heading (`##`) in the description.
+      - Link to the GitHub Release, binaries on IPNS, docker pull command and release notes in the description.
+    - [ ] Pin the topic globally so that it stays at the top of the category.
+    - [ ] If there is no more important banner currently set on Discourse (e.g. IPFS Camp announcement), make the topic into a banner.
+    - [ ] Check if Discourse post was automatically copied to:
+      - [ ] IPFS Discord #ipfs-chatter
+      - [ ] FIL Slack #ipfs-chatter
+      - [ ] Matrix
+  - [ ] Add a link from release notes to Discuss post (like we did here: https://github.com/ipfs/kubo/releases/tag/v0.15.0)
+  - [ ] Update the draft PR created for [interop](https://github.com/ipfs/interop) to use the new release and mark it as ready for review.
+  - [ ] Update the draft PR created for [IPFS Desktop](https://github.com/ipfs-shipyard/ipfs-desktop) to use the new release and mark it as ready for review.
+  - [ ] Update docs
+    - [ ] Run https://github.com/ipfs/ipfs-docs/actions/workflows/update-on-new-ipfs-tag.yml to generate a PR to the docs repo
+    - [ ] Merge the auto-created PR in https://github.com/ipfs/ipfs-docs/pulls ([example](https://github.com/ipfs/ipfs-docs/pull/1263))
+  - [ ] Get the blog post created and shared
+    - [ ] Submit a request for blog post creation using [the form](https://airtable.com/shrNH8YWole1xc70I).
+      - Notify marketing in #shared-pl-marketing-requests about the blog entry request (since the form tends to go to spam).
```

Comment:
```suggestion
      - Notify marketing in #shared-pl-marketing-requests about the blog entry request (since the form tends to go to spam; [example]([example](https://filecoinproject.slack.com/archives/C018EJ8LWH1/p1664885305374909))).
```